Formula/ko.rb

```diff
@@ -19,7 +19,7 @@ class Ko < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/google/ko/pkg/commands.Version=#{version}")
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/ko-build/ko/pkg/commands.Version=#{version}")
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/google/ko/pkg/commands.Version=#{version}")
```

For now, `ko`'s Go importpath is still `github.com/google/ko`, and we plan to move it to `ko.build` soonish. These paths are based on the importpath of the module, not the URL of the GitHub repo, so these need to be `google/ko` for now.

When we change the importpath and cut a release we'll want this to update to use the new importpath.

Reply:
This should still be Google for the time being